src/challenges.js

```diff
@@ -1,49 +1,167 @@
 // Desafio 1
-function compareTrue() {
+function compareTrue(firstValue, secondValue) {
   // seu código aqui
+  if (firstValue === true && secondValue === true) {
+    return true;
+  }
+  return false;
 }
 
 // Desafio 2
-function calcArea() {
+function calcArea(base, height) {
   // seu código aqui
+  return (base * height) / 2;
 }
 
 // Desafio 3
-function splitSentence() {
+function splitSentence(phrase) {
   // seu código aqui
+  let letter = ' '; let word = '';
+  let result = [];
+  for (let index = 0; index < phrase.length; index += 1) {
+    if (phrase[index] === letter) {
+      result.push(word);
+      word = '';
+    } else {
+      word += phrase[index];
+    }
+  }
+  result.push(word);
+  return result;
 }
 
 // Desafio 4
-function concatName() {
+function concatName(Array) {
```

Comment:
No javascript é uma boa prática escrever os nomes de variáveis/parâmetros começando minúsculo
```suggestion
function concatName(array) {
```

É importante alterar o restante dos `Array` nas linhas de baixo tb